bpf/complexity-tests/510/bpf_host/1.txt

```diff
@@ -35,3 +35,4 @@
 -DENABLE_SCTP=1
 -DENABLE_IPSEC=1
 -DEVENTS_MAP_RATE_LIMIT_PER_SECOND=1000
+-DENABLE_LOCAL_REDIRECT_POLICY
```

Comment:
I *think* that we've been declaring these as `-DENABLE_LOCAL_REDIRECT_POLICY=1`, because the `is_defined()` helper strictly requires 0 or 1 as value.

Reply:
Oh, I will fix it. Thanks for the comment!

Reply:
https://github.com/cilium/cilium/pull/35099